config/logger_test.go

```diff
@@ -0,0 +1,14 @@
+package config
```

Comment:
We can test the `InitLogger` function as-well:
![image](https://user-images.githubusercontent.com/12232750/108336057-50f66180-71dc-11eb-843b-00ce110c7780.png)